src/main/java/taxi/service/AuthenticationService.java

```diff
@@ -0,0 +1,8 @@
+package taxi.service;
+
+import taxi.exception.AuthenticationException;
+import taxi.model.Driver;
+
+public interface AuthenticationService {
```

Comment:
Move your all service's  impls to impl package and do the same with dao's impls.